src/data.js

```diff
@@ -13,6 +13,8 @@
 
 export const quizData = {
   currentQuestionIndex: 0,
+  questions_number: 3,
+  questionProgress: 0,
```

Comment:
Also don't need this... we have the exact same variable 2 lines above!

Reply:
I have already tried that but when I used `currentQuestionIndex` it affected `nextQuestion()`. But I am still working on it.